jitexecutor/jitexecutor-dmn/src/test/java/org/kie/kogito/jitexecutor/dmn/JITDMNServiceImplTest.java

```diff
@@ -162,6 +162,76 @@ void testEvaluationHitIds() throws IOException {
         Assertions.assertTrue(evaluationHitIds.contains(ruleId4));
     }
 
+    @Test
+    void testConditionalWithNestedDecisionTableFromRiskScoreEvaluation() throws IOException {
```

Comment:
Thanks @jomarko 
But this test is (semantically) already present: see
testConditionalExecutionIdsFromRiskScoreEvaluation
testEvaluationHitIds

Reply:
See my comment in PR adding model into drools repository.